modules/oauth-api/api-changes.json

```diff
@@ -1,7 +1,40 @@
 {
+  "4.3.0": {
```

Comment:
You'll likely need to change this later once it's cherry-picked.

Reply:
indeed